Documentation~/workflow-create-bezier.md

```diff
@@ -2,8 +2,18 @@
 
 The Bezier Shape tool uses a Bezier spline (curve) and extrudes along it to create a 3D version of the curve. You fine-tune the shape by using the tangent handles on the control points to bend the shape.
 
-![Default Bezier Shape](images/BezierShape_HeaderImage.png)
+![Default Bezier shape](images/BezierShape_HeaderImage.png)
 
-To create a Mesh based on a bezier (curve) shape, select ![Bezier Shape icon](images/icons/NewBezierSpline.png) **New Bezier Shape** from the [ProBuilder toolbar](toolbar.md). The initial curve has two control points with tangent handles to control the curve's bend. By default, ProBuilder creates the curve [in editing mode](bezier.md) so you can continue to reshape it.
+To define a Mesh based on a bezier (curve) shape:
 
-> **Tip:** You can access this tool from the ProBuilder menu (**Tools** > **ProBuilder** > **Editors** > **New Bezier Shape**).
+1. Make sure the [Experimental Features Enabled](preferences.md#experimental) preference is enabled (go to **Edit** > **Preferences** in Windows or **Unity** > **Preferences** in macOS from the main menu in Unity, then select the **ProBuilder** category from the list).
+
+2. Open the ProBuilder window (in Unity's top menu: **Tools** > **ProBuilder window**).
+
+	The [Edit Mode toolbar](edit-mode-toolbar.md) and the [ProBuilder toolbar](toolbar.md) appear.
+
+3. From the ProBuilder toolbar, click ![Bezier Shape icon](images/icons/NewBezierSpline.png) **New Bezier Shape**. 
+
+	> **Tip:** You can also access this tool from the ProBuilder menu (**Tools** > **ProBuilder** > **Editors** > **New Bezier Shape**).
+
+	The initial curve has two control points with tangent handles to control the curve's bend. By default, ProBuilder creates the curve [in editing mode](bezier.md) so you can continue to reshape it.
```

Comment:
Beziers shapes are referenced in both `Creating meshes` and `Experimental features` sections. Is this desirable ? Or should it be only in one place (I guess experimental one would make more sense)?

Reply:
Good point. I'll make the changes.